packages/block-template/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "block-template",
-  "version": "0.0.4",
+  "version": "0.0.5",
```

Comment:
We’ll need to publish the package after merging. Will you do this or shall I?

Reply:
I shall do it